File: Software engineering/project/backend/src/main/java/com/example/conferencemanagementsystem/model/Review.java
package com.example.conferencemanagementsystem.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Data
@Entity
@NoArgsConstructor
@AllArgsConstructor
public class Review {
    @Id
    @GeneratedValue
    private int id;
    private int grade; // from 1 to 7
    private String review;
    @OneToOne
    private Paper paper;
    @OneToOne
    private ProgramCommitteeMember programCommitteeMember;
}

File: Software engineering/project/webapp/src/app/model/review.ts
// todo

import {Paper} from "./paper";
import {ProgramCommitteeMember} from "./program-committee-member";

export class Review {
  id: number;
  grade: number;
  review: string;
  paper: Paper;
  programCommitteeMember: ProgramCommitteeMember;
}
